hedera-node/src/main/java/com/hedera/services/state/expiry/removal/TreasuryReturnHelper.java

```diff
@@ -74,6 +76,25 @@ void updateReturns(
 		}
 	}
 
+	EntityNumPair updateNftReturns(
+			final EntityNumPair nftKey,
+			final UniqueTokensListRemoval uniqueTokensRemoval
```

Comment:
Why not just pass `MerkleMap<EntityNumPair, MerkleUniqueToken> currUniqueTokens` here? There doesn't seem to be an advantage to having a `UniqueTokensListRemoval`.